instrumentation/rack/README.md

```diff
@@ -49,6 +49,23 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+### Request/response hooks
+
+You can pass hooks to extract additional attributes from the requests/responses.
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::Rack', {
+    request_hook: lambda { |span, env|
```

Comment:
can we link to the rack documentation on what the `env` argument is? Most users probably are not intimately familiar with rack internals.